hoomd/hpmc/pytest/test_shape_utils.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import numpy as np
+import pytest
+import coxeter
+from plato import geometry
+from pyhull.convex_hull import ConvexHull
```

Comment:
Tests should not introduce new dependencies. Run the needed code in a script and copy and past the out arrays needed into a hard-coded parameter structure to use for testing.

Reply:
Tests should not introduce new dependencies. Run the needed code in a script and copy and past the out arrays needed into a hard-coded parameter structure to use for testing.